plugins/wp-profiler/.wp-env.json

```diff
@@ -0,0 +1,11 @@
+{
+    "mappings": {
+        "wp-content/mu-plugins": "./"
+    },
+    "config": {
+        "WP_DEBUG": false,
+        "WP_DEBUG_LOG": false,
+        "WP_DEBUG_DISPLAY": false,
+        "SCRIPT_DEBUG": false
+    }
+}
```

Comment:
Having this file here seems a bit out of scope to me as it's for the environment, and the plugin folder itself does not make that environment work.

I think it would be great to implement a simple wp-env configuration in this repository for XHProf so that it's standalone. Of course for XHProf that requires us to first merge your PR https://github.com/WordPress/gutenberg/pull/48147.

My suggestion would be to, instead of a general `plugins` folder and this plugin, add a folder like `wp-profiling-env` which includes a basic `package.json` with the necessary dependencies and scripts to run `wp-env`, and then an `mu-plugins` folder with the `wp-profiler` plugin you have here, as that is most useful as part of that environment.

WDYT? We could of course split that work apart, since right now `@wordpress/env` does not yet support the necessary XHProf work, but I think eventually this would be a better outcome, have a standalone solution in here rather than only the MU plugin and require the rest to be set up manually.